trillian/migrillian/core/trillian.go

```diff
@@ -79,8 +80,7 @@ func NewPreorderedLogClient(
 	return &ret, nil
 }
 
-// getVerifiedRoot returns the current root of the Trillian tree. Verifies the
-// log's signature.
+// getRoot returns the current root of the Trillian tree.
 func (c *PreorderedLogClient) getVerifiedRoot(ctx context.Context) (*types.LogRootV1, error) {
```

Comment:
Forgot to rename the function?

Reply:
Ah yes! Re-did and rebased that PR so many times... 😛